.github/workflows/test_java_linux_cookbook.yml

```diff
@@ -39,10 +39,11 @@ jobs:
         java-version: [ '11', '17', '18' ]
         arrow_nightly: ['0', '1']
         exclude:
-          - java-version: '11'
-            arrow_nightly: '1'
-          - java-version: '17'
-            arrow_nightly: '1'
+          - arrow_nightly: '0'
+#          - java-version: '11'
+#            arrow_nightly: '1'
+#          - java-version: '17'
+#            arrow_nightly: '1'
```

Comment:
Why do we have commented-out code here?

Reply:
I thought we were changing how the cookbooks work so this shouldn't be an issue? Can we get this worked out properly?